.travis.yml

```diff
@@ -10,6 +10,7 @@ cache:
 branches:
   only:
     - develop
```

Comment:
The `branch -> only` array includes all branches that should trigger a travis build. Thus we want `develop` and now `master` for the upload.